client/pkg/testutil/leak.go

```diff
@@ -155,6 +155,7 @@ func interestingGoroutines() (gs []string) {
 				"rcrypto/internal/boring.(*PublicKeyRSA).finalize",
 				"net.(*netFD).Close(",
 				"testing.(*T).Run",
+				"crypto/tls.(*certCache).evict",
```

Comment:
```
=== RUN   TestVerifyHashKVAfterCompactMixVersion/#0
    leak.go:103: Found leaked goroutined BEFORE test appears to have leaked :
        sync.(*Map).LoadAndDelete(0xc00031e180, {0xe07320, 0xc00009fde0})
                /usr/local/go/src/sync/map.go:272 +0x192
        sync.(*Map).Delete(...)
                /usr/local/go/src/sync/map.go:297
        crypto/tls.(*certCache).evict(...)
                /usr/local/go/src/crypto/tls/cache.go:73
        crypto/tls.(*certCache).active.func1(0x0?)
                /usr/local/go/src/crypto/tls/cache.go:65 +0x67
```

The GC invokes the deletion. We should skip it.

Reply:
Please resolve this in a separate PR.